src/toy.rs

```diff
@@ -4,133 +4,125 @@ extern crate cranelift_simplejit;
 extern crate peg;
```

Comment:
Suggestions:
* These `extern crate`s can be converted to `use` by enabling Rust 2018.
* The file `toy.rs` belongs in `examples/` rather than `src/`, https://doc.rust-lang.org/cargo/guide/project-layout.html